packages/debugger/src/panels/breakpoints/index.ts

```diff
@@ -32,11 +30,12 @@ export class Breakpoints extends PanelWithToolbar {
     const body = new BreakpointsBody(model);
 
     this.toolbar.addItem(
-      'pause',
-      new CommandToolbarButton({
-        commands: commands.registry,
-        id: commands.pause,
-        label: ''
+      'pauseOnException',
+      new PauseOnExceptionsWidget({
+        service: service,
+        commands: commands,
+        icon: exceptionIcon,
+        tooltip: 'Pause on exception filter'
```

Comment:
This needs to be translated:

```suggestion
        tooltip: trans.__('Pause on exception filter')
```